data_loader.py

```diff
@@ -36,24 +37,37 @@ def tokenizer(text): # create a tokenizer function
     # Consider appending potentially named entity tags??
     return [tok.text for tok in spacy_en.tokenizer(text)]
 
+def genderToNum(gender):
+    assert(gender == 'm' or gender == 'f')
+    return 1 if gender == 'f' else 0
+
+def ageToInt(age):
+    return int(age[:-2])
```

Comment:
For the future I think you can just do like int(float(age)) possibly? I just ran into this too.